diffcp/cone_program.py

```diff
@@ -20,6 +20,19 @@ def pi(z, cones):
         [u, cone_lib.pi(v, cones, dual=True), np.maximum(w, 0)])
 
 
+def dpi_explicit(z, cones):
```

Comment:
nit: Rename to dpi_sparse_matrix?

Reply:
Done

Reply:
For consistency do you think we should do `_dense_matrix` everywhere too, instead of `dense`?

Reply:
Ah -- maybe `sparse` is better than `sparse_matrix`, then.